HISTORY.rst

```diff
@@ -41,6 +41,24 @@ Unreleased Changes (3.9)
     * ``convolve_2d`` keyword ``ignore_nodata`` to ``ignore_nodata_and_edges``.
     * ``get_raster_info`` / ``get_vector_info`` keyword ``projection`` to
       ``projection_wkt``.
+* Habitat Quality:
+    * Refactor of Habitat Quality that implements TaskGraph
+    * Threat files are now indicated in the Threat Table csv input under 
+    required columns: ``BASE_PATH``, ``CUR_PATH``, ``FUT_PATH``. 
```

Comment:
This is pretty trivial, but does the RST render properly here with the `required columns` line?  Would the indentation need to instead be like this?

```rst
    * Threat files are now indicated in the Threat Table csv input under 
      required columns: ``BASE_PATH``, ``CUR_PATH``, ``FUT_PATH``. 
```

Reply:
fixed here: https://github.com/natcap/invest/pull/75/commits/3986ed8ec5f431437802976f5f1fb3c1f0a45251